templates/nginx/nginx-config.yaml

```
@@ -28,7 +28,9 @@ data:
       log_format   main '$remote_addr - $remote_user [$time_local]  $status '
         '"$request" $body_bytes_sent "$http_referer" '
         '"$http_user_agent" "$http_x_forwarded_for" $http_x_scope_orgid';
+      {{- if .Values.nginx.config.verboseLogging }}
       access_log   /dev/stderr  main;
+      {{- else }}
```

```suggestion
      {{- end }}
```

Should this be `end` ?